sections/quick-order-form.liquid

```diff
@@ -21,17 +21,49 @@
   }
 {%- endstyle -%}
 
-{% comment %} TODO: enable theme-check once `line_items_for` is accepted as valid filter {% endcomment %}
-{% # theme-check-disable %}
-{%- assign items_in_cart = cart | line_items_for: product | sum: 'quantity' -%}
-{% # theme-check-enable %}
-
 <div class="color-{{ section.settings.color_scheme }} gradient">
   <quick-order-list
     id="{{ section.id }}"
     class="quick-order-form page-width section-{{ section.id }}-padding"
     data-id="{{ section.id }}"
   >
+    <div class="quick-order-form__search--wrapper">
+      <div class="field">
+        <input
+          class="search__input field__input"
+          id="Search-In-Template"
```

Comment:
This is an existing id for the Search template (in `main-search`). We shouldnt really be using the same id twice 👀 I know we dont allow this section in the search template, but it could be a problem in the future.

Can we rename it?

Reply:
oh, thats a good catch, I definitely missed it while I was coping it from the search. Thanks Sofia.

Reply:
Fixed 👌